### DISCUSSION TITLE
身份验证能否关闭或更改

### DISCUSSION THREAD

**Discussion by 401U** [zdz/ServerStatus-Rust#169] [Q&A]
现在使用的管理员登陆弹窗无法被我的密码管理软件（keepassxc）识别，能否更改为网页中输入账号密码类型的形式？

或者，能否提供一个将其彻底关闭的方法，之后可以使用 Cloudflare 的 Zero Trust 之类的服务来做访问控制？

**Comment by zdz** [comment]
后面加一下
